wavedroplet.js

```diff
@@ -167,7 +179,7 @@ function init(json) {
     });
 
     // TODO(katepek): Recalculate and redraw when resized
-    dim.height = (total_height - 3 * state.to_plot.length * dim.padding) / state.to_plot.length;
+    dim.height = (total_height - histHeight - (state.to_plot.length + 1) * dim.padding) / state.to_plot.length;
```

Comment:
This is now calculating the height correctly *except* it doesn't include the legend.  Maybe that's best, since we get more Y space for the useful part this way...